sympy/rubi/utility_function.py

```diff
@@ -65,3 +67,101 @@ def Sqrt(a):
 
 def ArcCosh(a):
     return acosh(a)
+
+def TogetherSimplify(expr):
```

Comment:
We need to discuss some of these functions I didn't get much information about them